metals/src/main/scala/scala/meta/internal/implementation/ImplementationProvider.scala

```diff
@@ -341,7 +341,7 @@ final class ImplementationProvider(
     }
   }
 
-  private def findSymbolInformation(
+  def findSymbolInformation(
```

Comment:
This should be private, or alternatively added to `Semanticdbs`.

Reply:
This will be needed in later PR's so I'll make it private again.